frontend/src/types.ts

```diff
@@ -1234,6 +1234,7 @@ export interface DashboardTemplateType {
     variables?: DashboardTemplateVariableType[]
     tags?: string[]
     image_url?: string
+    scope?: 'team' | 'everyone'
```

Comment:
I'm tempted to say "global" is better than everyone... but I'm also worried I'm nit-picking 🤣